go/pkg/log/server/property_test.go

```diff
@@ -447,11 +464,26 @@ func (suite *LogServerTestSuite) TestRecordLogDb_PushLogs() {
 						records, err = suite.lr.PullRecords(ctx, id.String(), 0, 1, time.Now().UnixNano())
 						suite.NoError(err)
 						if len(records) > 0 {
-							suite.Equal(int64(offset)+1, records[0].Offset)
+							suite.Equal(int64(offset)+1, records[0].Offset, "expected offset %d, got %d for collection id %s", int64(offset)+1, records[0].Offset, id)
 						}
 					}
 				}
 			},
+			"garbageCollection": func(t *rapid.T) {
+				// Garbage collect the model
+				suite.modelGarbageCollection(ctx, t)
+
+				// Garbage collect the SUT
+				err := suite.lr.GarbageCollection(ctx)
+				suite.NoError(err)
+				for id := range suite.model.CollectionData {
```

Comment:
should we make this an invariant to ensure that the result of GetCollection() == contents of the model?

Reply:
Will address in a follow up pR.